x/evm/core/vm/custom_eip_testing.go

```diff
@@ -6,14 +6,6 @@
 
 package vm
 
-import (
-	"fmt"
-	"sort"
-	"strings"
-
-	"golang.org/x/exp/maps"
-)
-
 var originalActivators = map[string]func(*JumpTable){
```

Comment:
Instead of hardcoding this one, we could write this:

```
var originalActivators = make(map[string]func(*JumpTable))

func init() {
	keys := GetActivatorsEipNames()

	originalActivators = make(map[string]func(*JumpTable), len(keys))

	for _, k := range keys {
		originalActivators[k] = activators[k]
	}
}
```